client/my-sites/domains/domain-management/dns/a-record.jsx

```diff
@@ -23,19 +23,20 @@ class ARecord extends Component {
 		const isDataValid = isValid( 'data' );
 		const isAaaaRecord = fieldValues.type === 'AAAA';
 
-		let namePlaceholder = translate( 'Enter subdomain (optional)', {
-			context: 'Placeholder shown when entering the optional subdomain part of a new DNS record',
+		let label = translate( 'Name (optional)', { context: 'DNS record' } );
+		if ( isAaaaRecord ) {
+			label = translate( 'Name (required)', { context: 'DNS record' } );
+		}
+
+		const namePlaceholder = translate( 'Enter subdomain', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'Enter subdomain (optional)', { context: 'Placeholder shown when entering the required subdomain part of a new DNS record'} )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/add-dns-record-page)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?